main.py

```diff
@@ -407,7 +420,12 @@ def main():
         args.read_only = False
     if not hasattr(args, "retry_sec"):
         args.retry_sec = 60
-
+    if not hasattr(args, "t_token") or args.t_token is None or not hasattr(args, "t_id") or args.t_id is None:
+        telegram_data = load_telegram_data("data/telegram.json")
+        if telegram_data is not None:
+            args.t_token=telegram_data["token"]
+            args.t_id=telegram_data["chatid"]
+            
```

Comment:
Bisher konnten wir uns immer zwischen zwei Varianten entscheiden: Entweder Einstellungen werden aus einer Datei (typischerweise `data/kontaktdaten.json`) geladen, oder per CLI-Option. Du möchtest hier offenbar beides unterstützen. Das führt aber zu einem Mehraufwand für die Wartung des Codes.

Bitte entscheide dich für eins von beiden: Entweder CLI-Optionen, oder Datei. Ich tendiere zu den CLI-Optionen. Falls Datei bevorzugt wird, sollte die vorhandene `kontaktdaten.json` entsprechend erweitert werden; denn sonst müssen wir uns für die zweite Datei eine Parallelstruktur zum Validieren dieser erstellen.

Falls es die CLI-Optionen werden: Diese sollten mit `None` initialisiert werden, da sie ja später (in `subcommand_search`) noch einmal auf `is not None` geprüft werden:

```suggestion
    if not hasattr(args, "telegram_token"):
        args.telegram_token = None
    if not hasattr(args, "telegram_id"):
        args.telegram_id = None            
```

Reply:
Per Datei das ganze zu laden macht für mich mehr sinn, ich würde mal davon ausgehen, das selbst wenn man mehrere vaccipy Instanzen laufen hat, man die Benachrichtigungen alle im selben Chat bekommen möchte

Reply:
Mit #351 braucht man nicht mehr mehrere Instanzen für eine Person. Allenfalls mehrere Instanzen für mehrere Personen.

Man kann das von mir aus auch gerne per Datei machen, aber wie wäre es, wenn es dann in die bestehende Kontaktdaten-Datei integriert wird? Spricht da etwas dagegen?